Underline Strikethrough.roboFontExt/lib/Underline StrikeThrough.py

```diff
@@ -200,14 +202,13 @@ def windowCloseCallback(self, sender):
     def drawLines(self, notification):
         glyph = notification["glyph"]
         if glyph:
-            fill(0)
+            fill(*self.lineColor)
             if self.underlinePosition[self.font.path] is not None and self.underlineThickness[self.font.path] is not None:
-                underlineY = self.underlinePosition[self.font.path] - self.underlineThickness[self.font.path] / 2
```

Comment:
Based on discussion, this line should stay as is, the position is the midpoint of the of the stroke, so it should be drawn this way

Reply:
Quick question: should fontmake be updating how it interprets this FontInfo value, then?

Reply:
Yes. It will need to change.

Reply:
Actually, it's in ufo2ft, I'll make a PR there once the UFO spec has been clarified

Reply:
Thank you!

Should
`underlineY = self.underlinePosition[self.font.path] - self.underlineThickness[self.font.path] / 2`
be
`underlineY = self.underlinePosition[self.font.path] - int(self.underlineThickness[self.font.path]) / 2`

to avoid floats?

Reply:
Yes